components/analytics/GoogleAnalytics.js

```diff
@@ -10,7 +10,7 @@ const GAScript = () => {
         src={`https://www.googletagmanager.com/gtag/js?id=${siteMetadata.analytics.googleAnalyticsId}`}
       />
 
-      <Script strategy="lazyOnload">
+      <Script strategy="lazyOnload" id="ga-script">
```

Comment:
Do we need to provide the `id` for the other `<Script />` as well?

Reply:
It's recommended for inline scripts.

> Inline scripts wrapped with <Script> require an id attribute to be defined to track and optimize the script.

The ones which I modified are those picked up by the linter.